cmd/kubernetes_create.go

```diff
@@ -22,20 +22,19 @@ var kubernetesCluster *civogo.KubernetesCluster
 var kubernetesCreateCmdExample = `civo kubernetes create CLUSTER_NAME [flags]
 
 Notes:
-* The '--create-firewall' It will be create the default rules for the firewall.
+* The '--create-firewall' will add default rules for the firewall if none are specified by using '--firewall-rules'.
```

Comment:
OK, so if you are changing the flag to require a list of ports, this can be confusing to the user. What are the default rules in this case?

Reply:
all port open

Reply:
Yeah, I think that needs to be clear in this (and all cases)